projects/novo-elements/src/elements/query-builder/expression-builder/expression-builder.component.ts

```diff
@@ -15,11 +15,13 @@ export class ExpressionBuilderComponent implements OnInit {
   public parentForm: AbstractControl;
   @Input() config: any;
   @Input() controlName: string;
+  @Input() orEnabled = false;
 
   constructor(private controlContainer: ControlContainer, private formBuilder: FormBuilder) {}
 
   ngOnInit() {
     this.parentForm = this.controlContainer.control;
+    setTimeout(() => this.addAndGroup());
```

Comment:
hmmmm...  well that may create an empty row of data that make saving a little harder?  maybe the implementor can add a row if they want.....

either way i think you can moove this to ngAfterViewChecked, maybe make the expression builder ChangeDetection onPush too.